src/background/Wallet/WalletRecord.ts

```diff
@@ -194,21 +199,32 @@ export class WalletRecordModel {
     return null;
   }
 
-  static getWalletByAddress(record: WalletRecord, address: string) {
-    for (const group of record.walletManager.groups) {
-      const wallet = group.walletContainer.getWalletByAddress(address);
-      if (wallet) {
-        return wallet;
+  static getWalletByAddress(
+    record: WalletRecord,
+    { address, groupId }: { address: string; groupId: string | null }
```

Comment:
Can we be in the situation when there are 2 same addresses in different groups? I thought, we remove readonly address when import it.